runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerOverlay.java

```diff
@@ -0,0 +1,79 @@
+package net.runelite.client.plugins.groundmarkers;
+
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Polygon;
+
+import java.util.ArrayList;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import net.runelite.api.Client;
+import net.runelite.api.Perspective;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayPriority;
+import net.runelite.client.ui.overlay.OverlayUtil;
+import javax.inject.Inject;
+import java.util.Iterator ;
+
+@Slf4j
+public class GroundMarkerOverlay extends Overlay
+{
+	@Inject
+	private Client client;
```

Comment:
Move this to constructor, you are also injecting client twise (also move rest of the injected fields to constructor)